client/package.json

```diff
@@ -45,5 +45,11 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
-  }
+  },
+  "description": "This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).",
```

Comment:
Interesting - Winnie was seeing the same thing! Do you know why it keeps getting recommitted?